plan/config_test.go

```diff
@@ -33,6 +34,8 @@ func TestReadConfigFromEnviron(t *testing.T) {
 	os.Setenv("AXIS_SERVER", "yarpc-go,yarpc-node")
 	os.Setenv("AXIS_TRANSPORT", "http,tchannel")
 	os.Setenv("BEHAVIOR_ECHO", "client,server,transport")
+	os.Setenv("CALL_TIMEOUT", "10s")
+	os.Setenv("HOST_TIMEOUT", "20s")
```

Comment:
Let's rename this to `WAIT_FOR_TIMEOUT` - this will make it more obvious that it relates to the `WAIT_FOR` setting.